.gitignore

```diff
@@ -44,3 +44,4 @@ tags
 .clangd/*
 compile_commands.json
 spack-*
+Offline
```

Comment:
I am curious why you need this?  Is it because of the symlink you need to add?

Reply:
Yes